PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs

```diff
@@ -88,6 +93,9 @@ private void Init()
         {
             try
             {
+                GetScreenSize();
+                InitSize();
+                InitPosition();
```

Comment:
can all these methods be merged into one?
and avoid those additional private members?